app/static/app/js/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class ErrorMessage extends React.Component {
+    constructor(props){
+        super();
+
+        this.state = {
+            error: props.message
+        };
+
+        this.close = this.close.bind(this);
+    }
+
+    close(){
+        this.setState({error: ""});
+    }
+
+    render(){
+        if (this.state.error !== ""){
+            return (
+                <div className="alert alert-warning alert-dismissible">
```

Comment:
Yeah I had seen the project some time ago, but decided to pass on it because of this: ` The project is under active development, and APIs will change.` and this https://github.com/react-bootstrap/react-bootstrap/wiki#100-roadmap (lots of stuff still a work in progress).